package.json

```diff
@@ -25,6 +25,7 @@
     "axios": "^1.2.0",
     "daisyui": "^2.42.1",
     "firebase": "^9.14.0",
+    "husky": "^8.0.3",
```

Comment:
husky should be in the dev dependency. please merge the main branch instead of applying changes manually

Reply:
It works like if we have something new in the main, you gonna just work on what you already have (on the version of main that your branch originates from). Only when you work is done, or you work needs the new PR, you save all commits, merge the main branch, then continue.
Under no circumstances should you apply the changes manually on your side.